framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1172,10 +1172,14 @@ protected void performFlowAnalysis(ClassTree classTree) {
 
                 while (!lambdaQueue.isEmpty()) {
                     Pair<LambdaExpressionTree, Store> lambdaPair = lambdaQueue.poll();
+                    MethodTree mt =
+                            (MethodTree)
+                                    TreeUtils.enclosingOfKind(
+                                            getPath(lambdaPair.first), Kind.METHOD);
                     analyze(
                             queue,
                             lambdaQueue,
-                            new CFGLambda(lambdaPair.first),
+                            new CFGLambda(lambdaPair.first, ct, mt),
```

Comment:
Can you use `classTree` instead of `ct`? And open an issue that we should look into which one of the two we should be using here. It is definitely a bit odd to use both within the same expression.